src/app-azure.js

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SharedMap } from "fluid-framework";
+import { AzureClient, InsecureTokenProvider } from "@fluidframework/azure-client";
+import { vueRenderView as renderDiceRoller } from "./view";
+
+// This configures the AzureClient to use a local in-memory service called Tinylicious.
+// You can run Tinylicious locally using 'npx tinylicious'.
+const localConfig = {
+    tenantId: "local",
+    tokenProvider: new InsecureTokenProvider("anyValue", { id: "userId" }),
+    // if you're running Tinylicious on a non-default port, you'll need change these URLs
+    orderer: "http://localhost:7070",
+    storage: "http://localhost:7070",
+};
+
+// This configures the AzureClient to use a remote Azure Fluid Service instance.
+// const azureUser = {
+//     userId: "Test User",
+//     userName: "test-user"
+// }
+
+// const prodConfig: AzureConnectionConfig = {
+//     tenantId: "",
+//     tokenProvider: new AzureFunctionTokenProvider("", azureUser),
+//     orderer: "",
+//     storage: "",
+// };
+
+const client = new AzureClient(localConfig);
+const containerConfig = {
+    initialObjects: { dice: SharedMap }
```

Comment:
This code doesn't run. You need `diceMap` here or `dice` below

```suggestion
    initialObjects: { diceMap: SharedMap }
```